extensions/notebook/src/dialog/configurePythonDialog.ts

```diff
@@ -59,7 +59,7 @@ export class ConfigurePythonDialog {
 		this.dialog.cancelButton.label = this.CancelButtonText;
 		this.dialog.cancelButton.onClick(() => {
 			if (rejectOnCancel) {
-				this.setupComplete.reject(localize('configurePython.pythonInstallDeclined', "Python installation was declined."));
+				this.setupComplete.reject('INFO: '.concat(localize('configurePython.pythonInstallDeclined', "Python installation was declined.")));
```

Comment:
I'd look into passing a separate field along with the severity or something. Possibly change it so cancel resolves it but with a param cancelled = true or something.